modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -0,0 +1,84 @@
+While some might use the Connected Apps feature to build CI/CD pipelines, others can productize additional third-party use-cases on top of Anypoint Platform.
+
+In addition to enabling new flows to authenticate users, developers can enable or disable clients and view client usage.
+For example, developers can see how many organizations are using the app and how many users have access.
+
+The Connected Apps feature supports OAuth 2.0 and OpenID Connect. As such, developers must ensure their app can use tokens to request access to user data.
+To find out what specific OpenID Connect capabilities are supported, refer to the information provided in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.well-known/openid-configuration[discovery endpoint].
+All scopes in the endpoint map directly to permissions available in Anypoint Platform, with some exceptions noted in the *Endpoint Scopes* section.
+
+To foster a consistent authentication experience for end users, third-party developers with their own websites or web apps can allow users to sign in using the *Sign in with Anypoint Platform* option.
+
+Developers who have the organization administrator role can create apps and specify access scopes in Anypoint Platform.
+
+== Token Flow
+
+When a developer creates an app, the app requests access to user data.
+When an end user receives such a request and authorizes the app to access their data, the app receives an access token, which allows the app to call Anypoint Platform on the user’s behalf.
+The access token allows the app to see who the user is by using the user’s Anypoint Platform credentials.
+
+== Endpoint Scopes
+
+Most of the scopes in the https://anypoint.mulesoft.com/accounts/api/v2/oauth2/.well-known/openid-configuration[discovery endpoint] map directly to existing permissions in Anypoint Platform. Other available scopes are defined as follows:
+
+* `full`: Full access to anything the user can do.
+* `read:full`: Read-only access to anything the user can read.
+* `openid`: Read-only access to the user's username and unique Anypoint ID.
+* `profile`: Read-only access to the user's Anypoint profile.
+* `email`: Read-only access to the user's email.
+* `offline_access`: Access to the user's data when they are not logged in.
+
+== OIDC Endpoints and Tokens
+
+To work with OpenID Connect, apps that require access to user data when the user isn’t logged in have to exchange refresh tokens for new access tokens.
+Refresh tokens expire after 90 days. Access tokens expire after the user's session expires. If the app is using the authorization code grant type, the `offline_access` scope is required to use refresh tokens after original access tokens expire.
+To request a new access token, you must send a request to the token endpoint using the `refresh_token` grant type and the `offline_access` scope.
```

Comment:
Let's remove "and the offline_access" scope from here. We don't show it in the example below and it's not strictly necessary.

Reply:
```suggestion
To request a new access token, you must send a request to the token endpoint using the `refresh_token` grant type.
```